dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -122,8 +123,15 @@ def check_import(self, lib_path, dali_stub):
 
             try:
                 print("Loading DALI TF library: ", lib_path_tmpdir)
-                tf.load_op_library(lib_path_tmpdir)
-                return True
+                # try in a separate process just in case it recives SIGV
+                p = Process(target=tf.load_op_library, args=(lib_path_tmpdir, ))
```

Comment:
Just a question, do the exceptions propagate somehow through the Process, or we just basically look at the exit code now?

Reply:
They don't out of the box. I think in this case exception from the worker process only results in non zero exit code.

Reply:
I checked:
```
from multiprocessing import Process

def my_f():
  1/0

p = Process(target=my_f)
p.start()
p.join()
p.exitcode
```
It shows 1. Other exceptions, like `StopIteration` also shows 1. I think the change covers this case.